### ISSUE TITLE
Could not load file or assembly FastMember.Signed, Version=1.0.0.9 with ClosedXML

### ISSUE THREAD

**Issue by blfuentes** [mgravell/fast-member#50]
After updating to version 1.2.0 I get the following error when using ClosedXML:
`Could not load file or assembly 'FastMember.Signed, Version=1.0.0.9, Culture=neutral, PublicKeyToken=9e8f22703bef9a29' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)`

Prior version 1.1.0 worked as expected.

**ClosedXML v0.91.0**

**Comment by mgravell** [comment]
Yeah, apparently I screwed up something when I changed build tools. Will be
fixed.
